test/corpus/let_bindings.txt

```diff
@@ -257,11 +257,11 @@ and b = c
         (decorator (decorator_identifier))
         (decorator (decorator_identifier))
         (value_identifier)
-        (value_identifier))
+        (value_identifier)
       (let_binding
         (decorator (decorator_identifier))
         (value_identifier)
-        (number)))))
+        (number))))))
```

Comment:
The third `let_binding` is not a child of the second `let_binding`. So, would you indent this block one step right to make this apparent?